services/nginz/Makefile

```diff
@@ -53,15 +56,19 @@ default: compile
 clean:
 	-rm -rf src $(DIST) .metadata zwagger-ui/swagger-ui
 
+.PHONY: check-deps
+check-deps:
+	PKG_CONFIG_PATH=$(EXTRA_PKG_PATH) pkg-config --exists libzauth || { echo -e "\n\033[0;31m    The 'libzauth' library was not found\033[0m\n    pkg-config path = $(EXTRA_PKG_PATH)\n\n    Suggestion: run 'make libzauth' on the top level\n"; exit 1; }
+
```

Comment:
How about adding another target here:
```Makefile
.PHONY: libzauth
libzauth:
	$(MAKE) -C ../../libs/libzauth install
```
And, instead of printing a warning in the `|| { ..` block, doing a `|| { echo -e "The 'libzauth' library was not found, attempting to install it..." && make libzauth` ?

Reply:
Sounds ok.